### PR TITLE
Add some form of telling people it's dead.

### LINKED ISSUE #51: Moving items around breaks auto-advance order
Reported on Slack: https://ury.slack.com/archives/C011DBCL36U/p1586459805045200

Repro: https://www.dropbox.com/s/5j6h0qol22ayav9/AutoAdvanced.mov?dl=0 https://www.dropbox.com/s/s2ai6yo03en6cxr/AutoAdvance%28Backwards%29.mov?dl=0

### PR DESCRIPTION
**Pull request by mstratford**
Gives people who can't / won't see Timelord during their show a bit of an idea how a show is going on.

Fixes #51

Todo:
~~Work out how to determine we went Jukebox due to dead air #93~~ (EDIT: I think i'll do this in the next release, along side some stream health monitoring stuff.)

### CODE REVIEW

**Comment by mstratford** [comment]
I guess i'm now happy with this, and this will likely be the last of the PR work for WebStudio 1.5.0.

File: src/navbar/timelord.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    return { id: studio, name: sourceName };
+  }
+
+  async function getSilence() {
+    let silence = await myradioApiRequest("/selector/issilence", "GET", null);
+
+    return silence;
+  }
+
+  const broadcastState = useSelector((state: RootState) => state.broadcast);
```
Comment (markspolakovs) [inline src/navbar/timelord.tsx:38]: Can just be `state.broadcast.stage` to avoid over-scoping

**Review by markspolakovs — APPROVED** [review]
Looks good, modulo nitpick

**Comment by mstratford** [comment]
Woops, didn't mean to dismiss that.

**Review by markspolakovs — APPROVED** [review]

**Pull request merged**